### ISSUE TITLE
missing my eth in wallet, which was sent to it

### ISSUE THREAD

**Issue by subdigger** [ethereum/mist#3613]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
version: 0.9.3
os: Linux linux-sjzh.site 3.16.7-53-desktop #1 SMP PREEMPT Fri Dec 2 13:19:28 UTC 2016 (7b4a1f9) x86_64 x86_64 x86_64 GNU/Linux
```
wallet was downloaded from https://github.com/ethereum/mist/releases

I can not see ethereum that was send from pool.
Also tried mist with the same result.

According to looooooong time of synchronizing, was found that if i delete all blocks data - it can synchronize more faster. but also no success.

![image](https://user-images.githubusercontent.com/4728889/35234771-2a3b1a3e-ff9a-11e7-8c3e-085573826b42.png)

transaction https://etherscan.io/tx/0x33c1a2d8a4825b29cb9bd667d0aa010e1123576f7f7b1c4f6cfe631616b74b06

current sync state (was already synced before)
![image](https://user-images.githubusercontent.com/4728889/35233916-657002ca-ff97-11e7-9e94-f4bf8544aa44.png)

**Comment by ryanio** [comment]
Hey, sorry about the sync issues. Please refer to this issue here with some more information and ways to help: https://github.com/ethereum/mist/issues/3097

**Comment by subdigger** [comment]
@ryanio saw that before. already digg internet about how to....

as i wrote i have already remove db and synchronize again (twice, including light update)

the min reason is not that it sync too log. but that i do not see the money....

**Comment by subdigger** [comment]
even if synchronization not finished. transaction is 6 days old. so it actually should`n be in last transactions

**Comment by Kolya56** [comment]
same issue here. sent eth to so-called "ethereum" wallet Jan.18. ethereum doesn't even let me create a wallet in the first place. no trace of the deposit past etherscan...is this a scam?

**Comment by Kolya56** [comment]
"syncing nodes" is like watching grass grow lol

**Comment by cryptojunky** [comment]
@subdigger The balance usually does not update till wallet is fully synced. Also try restarting your pc and open wallet with sync mode light. Patience is the key here, The Devs are working on a fix. It will sync though. Run it and go have a cup of coffee or maybe a whole pot.

@Kolya56 do you have the transaction ID? Where did you send it from and to?

**Comment by subdigger** [comment]
@cryptojunky thanks for advice, but:
- already done that triple on 2 different computers (linux, going to try windows :\ )
- barrel of coffee does not helps
- it already synced to that transaction, i think, cause it was 2 weeks ago.
- actually wallet newer-sync-to-100% it always have 200-500 blocks not synced

**Comment by Linolotzin** [comment]
i have the same problem. and noone is helping here! how can you release a product with such a major flaw. I need my funds.. can some pls take a look at my thread? :( or solve this one..

**Comment by Taledeyo** [comment]
me too..i don't get notthing.

**Comment by subdigger** [comment]
@ryanio whan we may expect of fixing this problem?
and is there any way to get money out to some other wallet ?

**Comment by ryanio** [comment]
@subdigger Sorry we are working on the sync issues on the dev team for the next release. If you want to do something more immediately, you can try using https://myetherwallet.com

**Comment by subdigger** [comment]
@ryanio thanks that service helps.

**Comment by evertonfraga** [comment]
Hi @subdigger, we've released version 0.10.0, and it's much easier to synchronize. Please give it a try :)
